doc/isomorph.xml

```diff
@@ -54,6 +54,9 @@
     the value of either <C>BlissAutomorphismGroup</C> or  <Ref
       Attr="NautyAutomorphismGroup"/>. These groups are, of course, equal but
     their generating sets may differ. <P/>
+    The attribute <Ref Attr="AutomorphismGroup" Label="for a digraph,
+      homogeneous list, and list" /> returns the value of
+    <C>BlissAutomorphismGroup</C> as it is not implemented for &nauty;
```

Comment:
Perhaps say that that is where you can find the requirements of the optional arguments to this function?

Also, the three-argument version of `BlissAutomorphismGroup` is not an attribute, so I think it should be documented more like:
```suggestion
  <Attr Name="BlissAutomorphismGroup" Arg="digraph"/>
  <Oper Name="BlissAutomorphismGroup" Arg="digraph, vertex_colours, edge_colours"/>
```
I presume - or something like that (perhaps with a two-argument version, if applicable, as mentioned above). Perhaps they will need labels.

Reply:
Agreed and fixed